sdk/jdbc/azure-identity-providers-jdbc-mysql/README.md

```diff
@@ -143,6 +147,14 @@ This picture shows how the jdbc authentication plugin for MySQL authenticating w
 
 ![mysql_design.png](img/mysql_design.png)
 
+1. The JDBC auth plugin will get an accesstoken from Azure AD.
+2. The JDBC driver will take the token obtained from step 1 as the password ( `token as password`) to connect with the MySQL server.
+3. The MySQL server will check the accesstoken and authenticate internally.
+
+### Token as password
+
+Instead of using a password directly, using a token as the password is recommended. Access tokens can have restricted permissions and expiration times. From a security point of view, using shorter-lived access tokens is much safer than using a password.
```

Comment:
```suggestion
Instead of using a password directly, using a token as the password is recommended. Access tokens can have restricted permissions and expiration time. From a security point of view, using shorter-lived access tokens is much safer than using a password.
```